src/controller/config.tsx

```diff
@@ -48,14 +48,6 @@ const SNOOZE_OPTIONS: { [key in SnoozeReason]: SnoozeOption } = {
     type: "LINK",
     subType: "troubleticket"
   },
-  bcu: {
```

Comment:
Ironically, one of the first things we thought we had learned about what the users were doing... 🤣